app/controllers/groups_controller.rb

```diff
@@ -2,7 +2,7 @@ class GroupsController < ApplicationController
   before_action :set_group, only: [:show, :edit, :update, :destroy]
   before_action -> { unauthorized_user(@group.owner) }, only: [:update, :edit, :destroy]
 
-  def index
+  def index # Delete for the future?
     @groups = Group.all
   end
```

Comment:
why can everyone see this?

Reply:
Mostly debugging purposes to view all the groups at once. I will remove it in this branch.